### PR TITLE
add Transaction

### LINKED ISSUE #5: Web server: transaction database model
The server must store information about financial transactions. For now, we would like to store the following:
- a timestamp
- a person's name who made the transaction (this will later be expanded to double-entry accounting)
- the transaction amount in USD
- the receipt file for reference

### PR DESCRIPTION
**Pull request by orogenic**
resolve #5

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,12 +1,47 @@
...
+)
+COLLATE='utf8mb4_general_ci'
+ENGINE=InnoDB
+;
+"""
 
 app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+mysqldb://{}:{}@localhost/{}?charset=utf8mb4'.format(
+    MYSQL_USERNAME, MYSQL_PASSWORD, MYSQL_DATABASE)
+app.config['SECRET_KEY'] = '91caba99a921ff7ac48097b56c6d9f6c'
```
Comment (orogenic) [inline app.py:31]: unused, remove

File: app.py [hunk 2]
```diff
@@ -1,12 +1,47 @@
+# vim: set ts=4 sw=4 expandtab:
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.dialects import mysql
+
+MYSQL_USERNAME = 'developer'
+MYSQL_PASSWORD = '9b338b88b9e0774846a720843f9ba632'
+MYSQL_DATABASE = 'development'
```
Comment (orogenic) [inline app.py:8]: read these 3 from environment

File: app.py [hunk 3]
```diff
@@ -1,12 +1,47 @@
+# vim: set ts=4 sw=4 expandtab:
```
Comment (orogenic) [inline app.py:1]: configure developer vimrc, remove modeline

File: app.py [hunk 4]
```diff
@@ -1,12 +1,47 @@
...
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.dialects import mysql
```
Comment (orogenic) [inline app.py:4]: unused import

File: requirements.txt [hunk 5]
```diff
@@ -1,7 +1,11 @@
...
 Flask==1.1.1
+Flask-SQLAlchemy==2.4.4
 gunicorn==20.0.4
 itsdangerous==1.1.0
 Jinja2==2.11.1
 MarkupSafe==1.1.1
+mysqlclient==2.0.1
+SQLAlchemy==1.3.19
 Werkzeug==1.0.1
+WTForms==2.3.3
```
Comment (orogenic) [inline requirements.txt:11]: unnecessary, remove

**Review by jonathan-mejia-deluna — APPROVED** [review]
No comment

**Review by anguyen120 — APPROVED** [review]

**Pull request merged**